README.md

```diff
@@ -10,3 +8,46 @@ To use sbt-npm-dependencies in an existing SBT project with Scala 2.12 or a late
 ```scala
 addSbtPlugin("io.chrisdavenport" %% "sbt-npm-dependencies" % "<version>")
 ```
+
+In your `build.sbt`, enable it with `enablePlugins(NpmDependenciesPlugin)` and declare your dependencies with
+
+```scala
+Compile / npmDependencies ++= Seq(
+  "npmPackageName" -> "version",
+  "secondNpmPackageName" -> "version",
+  ...
+)
+```
+
+## Internals
+
+(For integration with other build systems)
+
+sbt-npm-dependencies will export a file called `NPM_DEPENDENCIES` containing the dependencies in the following format:
+
+```js
```

Comment:
```suggestion
```json
```

Reply:
I used `js` so it doesn't scream on the `...`.